a3/terraform/modules/cluster/mig/variables.tf

```diff
@@ -55,6 +55,22 @@ variable "resource_prefix" {
   type        = string
 }
 
+variable "machine_type" {
```

Comment:
shouldn't this be a part of variable `instance_groups` similar to gke's `node_groups`